dev/tests/api-functional/testsuite/Magento/GraphQl/Customer/CreateCustomerTest.php

```diff
@@ -237,4 +237,19 @@ public function testCreateCustomerIfPassedAttributeDosNotExistsInCustomerInput()
 QUERY;
         $this->graphQlQuery($query);
     }
+
+    public function tearDown()
+    {
+        /** @var \Magento\Framework\Registry $registry */
+        $registry = Bootstrap::getObjectManager()->get(\Magento\Framework\Registry::class);
+        $registry->unregister('isSecureArea');
+        $registry->register('isSecureArea', true);
+        $customersCollection = Bootstrap::getObjectManager()->create(\Magento\Customer\Model\Customer::class);
```

Comment:
Looks like you should operate with customer collection but not with customer object

Reply:
Changes were done